cluster/swarm/cluster.go

```diff
@@ -14,6 +14,7 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/docker/pkg/discovery"
 	"github.com/docker/docker/pkg/stringid"
+	apitypes "github.com/docker/engine-api/types"
```

Comment:
If you don't mind, could we skip the name `apitypes`? It's not needed in this package I think, and conflicts with my PR.